DependencyInjection/DoctrineExtension.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Doctrine\Bundle\DoctrineBundle\DependencyInjection;
 
+use Doctrine\Bundle\DoctrineBundle\Command\Proxy\PingCommand;
```

Comment:
The class in the bundle will always exist. The class_exists check should be on the DBAL class.